source/rust_verify/example/invariants.rs

```diff
@@ -0,0 +1,169 @@
+// rust_verify/tests/example.rs expect-failures
+
+#[allow(unused_imports)]
+use builtin::*;
+mod pervasive;
+use pervasive::*;
+use crate::pervasive::{invariants::*};
+
+/*
```

Comment:
I'd suggest adding a note about why this code and some of the code later on is commented out. If it's not relevant anymore, consider removing it.

Reply:
Tests instead of the example for failure cases, as discussed in the other comment, if you can. Otherwise just add a comment to say why they're commented out. Not a big deal, though.